src/reconcile/rec_visibility.c

```diff
@@ -281,6 +282,15 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
         if (upd->start_ts > max_ts)
             max_ts = upd->start_ts;
 
+        /*
```

Comment:
Can you add some more details in the comment as this update is not stored in the data store but it will be saved in the history store?